stl/inc/memory

```diff
@@ -3247,26 +3267,12 @@ public:
         return false;
     }
 
-    void wait(shared_ptr<_Ty> _Old, memory_order = memory_order::seq_cst) const noexcept {
-        for (;;) {
-            auto _Rep   = this->_Lock_and_load();
-            bool _Equal = this->_Ptr.load(memory_order::relaxed) == _Old._Ptr;
-            this->_Store_and_unlock(_Rep);
-            if (!_Equal) {
-                break;
-            }
-            __std_atomic_wait_direct(&this->_Ptr, &_Old._Ptr, sizeof(_Old._Ptr), _Atomic_wait_no_timeout);
-        }
-    }
-
-    void notify_one() noexcept {
-        this->_Ptr.notify_one();
+    void wait(shared_ptr<_Ty> _Old, memory_order _Order = memory_order_seq_cst) const noexcept {
+        this->_Wait(_Old._Ptr, _Order);
     }
 
-
-    void notify_all() noexcept {
-        this->_Ptr.notify_all();
-    }
+    using _Base::notify_all;
+    using _Base::notify_one;

```

Comment:
I observe that the base class, and the Standard, declares `notify_one` before `notify_all`, not that this really matters. Occurs below. No change requested.

Reply:
clang format wants them in the wrong order

Reply:
Oh interesting, I didn't realize it sorted using-declarations. Thanks.